src/main/java/teammates/logic/core/FeedbackResponseCommentsLogic.java

```diff
@@ -79,22 +80,23 @@ public FeedbackResponseCommentAttributes getFeedbackResponseComment(Long feedbac
         return frcDb.getFeedbackResponseComment(feedbackResponseCommentId);
     }
     
-    public FeedbackResponseCommentAttributes getFeedbackResponseComment(
-            String responseId, String giverEmail, Date creationDate) {
+    public FeedbackResponseCommentAttributes getFeedbackResponseComment(String responseId, String giverEmail,
+                                                                        Date creationDate) {
         return frcDb.getFeedbackResponseComment(responseId, giverEmail, creationDate);
     }
 
-    public List<FeedbackResponseCommentAttributes> getFeedbackResponseCommentForResponse(String feedbackResponseId){
+    public List<FeedbackResponseCommentAttributes> getFeedbackResponseCommentForResponse(String feedbackResponseId) {
         return frcDb.getFeedbackResponseCommentsForResponse(feedbackResponseId);
     }
     
     public List<FeedbackResponseCommentAttributes> getFeedbackResponseCommentForSession(String courseId,
-            String feedbackSessionName) {
+                                                                                        String feedbackSessionName) {
```

Comment:
Can shorten this line? After you create the pull request, view the diff in GitHub to see if any line needs horizontal scrolling to view.